optionals/particles/script_component.hpp

```diff
@@ -51,5 +51,5 @@
 
 #define CLOUDLET_UNDERWATER(color) class SmokeShell##color##UW; \
 class SmokeShell##color##2UW: SmokeShell##color##UW { \
-    postEffects = QUOTE(ACE_SmokeAfterWater##color); \
+    postEffects = QUOTE(DOUBLES(ACE_SmokeAfterWater,color)); \
```

Comment:
```suggestion
    postEffects = 'ACE_SmokeAfterWater##color'; \
```

Reply:
Tried this, now resolves in-game as
`type = "ACE_SmokeAfterWaterWhite";` & `postEffects = "'ACE_SmokeAfterWaterWhite'";`

Same error pops when testing though.
> 20:53:32 Warning Message: No entry 'bin\config.bin.'ACE_SmokeAfterWaterWhite''.

Reply:
```suggestion
    postEffects = ACE_SmokeAfterWater##color; \
```